app/src/main/java/com/nutomic/syncthingandroid/activities/DeviceActivity.java

```diff
@@ -149,6 +151,17 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
         }
     };
 
+    private final CompoundButton.OnCheckedChangeListener mDevicePausedCheckedChangeListener = new CompoundButton.OnCheckedChangeListener() {
+        @Override
+        public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
+            if (mDevice.paused != isChecked) {
+                mDeviceNeedsToUpdate = true;
+
+                mDevice.paused = isChecked;
+            }
```

Comment:
Noop blank line.
Also can't this just be:
```
public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
  mDeviceNeedsToUpdate = true;
  mDevice.paused = isChecked;
}
```

Reply:
I simplified the code by merging the button change part in mCheckedListener (as its already the case in FolderActivity.java)